recipes/jupyter_qtconsole_colorschemes/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jupyter_qtconsole_colorschemes" %}
+{% set version = "0.8.1" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "26e12e926be13c2f4e8290b924ee8223d4989fb65c83c6af7197ce3be231d451" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  preserve_egg_dir: True
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
```

Comment:
It looks like `pygments` is required. Also is `setuptools` a runtime requirement?